firmware/application/apps/ui_encoders.hpp

```diff
@@ -66,12 +66,14 @@ class EncodersConfigView : public View {
 	
 	Labels labels {
 		{ { 1 * 8, 0 }, "Type:", Color::light_grey() },
-		{ { 16 * 8, 0 }, "Clk:", Color::light_grey() },
-		{ { 25 * 8, 0 }, "kHz", Color::light_grey() },
-		{ { 14 * 8, 2 * 8 }, "Frame:", Color::light_grey() },
-		{ { 26 * 8, 2 * 8 }, "us", Color::light_grey() },
-		{ { 2 * 8, 4 * 8 }, "Symbols:", Color::light_grey() },
-		{ { 1 * 8, 11 * 8 }, "Waveform:", Color::light_grey() }
+		{ { 1 * 8, 2 * 8 }, "Clk:", Color::light_grey() },
+		{ { 10 * 8, 2 * 8 }, "kHz", Color::light_grey() },
+        { { 17 * 8, 2 * 8 }, "Step:", Color::light_grey() },
```

Comment:
Can you please indent that line so it fits with the other lines around it ?
After this I'll merge your changes

Reply:
I did it by editing online.